app/views/course/achievements/_achievement.html.slim

```diff
@@ -1,27 +1,38 @@
-= content_tag_for(:tr, achievement,
-                  class: draft_class(achievement) + \
-                         achievement_status_class(achievement, current_course_user)) do
+table.table.achievement-list.table-hover
```

Comment:
by convention a singular partial renders only one item, with the name of the variable being the same as the partial. Make this plural.